### PR TITLE
modify footer ~add links, and changed copyright label

### LINKED ISSUE #2881: Docusaurus footer needs to be updated
Our Docusaurus footer needs some tender loving care.

2022-02-11 09_59_53-telescope - Visual Studio Code

Specifically:
- The copyright should be `Copyright (c) 2022 Telescope, Seneca's Centre for Development of Open Technology. Built with Docusaurus.`
- We don't have Stack Overflow, Discord, Twitter

I liked [Supabase](https://supabase.com/)'s footer so we can use theirs as a template
| Product | Resources | Developers | 
| --- | --- | --- | 
| Telescope Website | System Status | Documentation |
| Telescope Mobile Apps | Terms of Service | Slack | 
| | Planet CDOT Feeds | GitHub |
| | | Staging Server |

This is just a suggestion for our footer, if you feel there's a better way to organize links or something is missing or something doesn't need to be included, comment below :)

### PR DESCRIPTION
**Pull request by sancodes**
## Issue This PR Addresses
Docusaurus footer needs to be updated, fixes #2881

## Type of Change

- [ ] **Bugfix**: Change which fixes an issue
- [X] **New Feature**: Change which adds functionality
- [ ] **Documentation Update**: Change which improves documentation
- [ ] **UI**: Change which improves UI

## Description
Modify the Footer:
- Added new labels for links, that successfully directs to proper site when clicked.
- Changed Footer's Copyright label
- Changes made passes local test, with screenshot attached below and no updates on documentation as the documentation is still being worked on at the moment. 
Screen Shot 2022-02-11 at 5 11 22 PM

## Steps to test the PR

With Environment Set up to direction to see where the change has been added: 
- Clone the Telescope repository
- Install pnpm globally: npm install -g pnpm
- Install the dependencies: pnpm install (this will also install the dependencies in the Docusaurus website)
- Change directory to src/docs (this is where all the code lives for our Docusaurus website): cd src/docs
- Start the app: pnpm start
- Visit localhost:4631 in your browser.
- Scroll all the way to the bottom of the page, to where the Footer is. 
- You will be able to see various labels that directs to its sites. 
- You can also see updated Footer Copyright label. 

## Checklist

- [X] **Quality**: This PR builds and passes our npm test and works locally
- [X] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [X] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not (if applicable)
- [X] **Documentation**: This PR includes updated/added documentation to user exposed functionality or configuration variables are added/changed or an explanation of why it does not(if applicable)

### CODE REVIEW

**Comment by TueeNguyen** [comment]
Can you rebase instead of merging? You can reset your branch to the commit before your 1st merge commit and do:
1. update your fork 
2. `git checkout master`
3. `git pull origin master`
4. `git checkout footer-enhancement`
4. `git rebase master`

If you have further trouble with rebasing, I'm willing to help

**Review by TueeNguyen — CHANGES_REQUESTED** [review]
@sancodes Can you rebase instead of merging? You can reset your branch to the commit before your 1st merge commit and do:
1. update your fork 
2. `git checkout master`
3. `git pull origin master`
4. `git checkout footer-enhancement`
4. `git rebase master`

If you have further trouble with rebasing, I'm willing to help

**Comment by sancodes** [comment]
@TueeNguyen 

so I ran 
1. git reset --hard f6ea868e7f65b32dae5d2d8118463bb6f3db32e1 
2. git checkout master
3. git pull origin master
4. git checkout footer-enhancement
5. git rebase master
6. git push origin footer-enhancement

Then while pushing I get this error. 

Screen Shot 2022-02-12 at 3 26 32 PM


Maybe I did something wrong. Can you help me on this? thank you kindly.

**Comment by TueeNguyen** [comment]
You're doing it correctly, the SHA commit after you rebase is different than the one you have in this PR, so just do a `git push origin footer-enhancement -f`

This way you're gonna force push it

**Comment by sancodes** [comment]
@TueeNguyen 

Seems like it worked, Thank you.

Did that meet your requirement?

**Comment by TueeNguyen** [comment]
I just checked your fork, you haven't `fetch and merge` your fork. Update it and do the steps again without doing `git reset`

**Comment by sancodes** [comment]
@TueeNguyen 
I just went to the forked version of telescope on my github, clicked on the button that said, 'fetch and merge'
then followed the steps. 

I dont know if I messed up anything. Please let me know, if everything is working fine.

File: src/docs/docusaurus.config.js [hunk 1]
```diff
@@ -62,42 +62,60 @@ const config = {
...
               },
+              {
+                label: 'Staging Server',
+                href: 'https://dev.telescope.cdot.systems/',
+              },
             ],
           },
         ],
-        copyright: `Copyright © ${new Date().getFullYear()} My Project, Inc. Built with Docusaurus.`,
+        copyright: `Copyright © ${new Date().getFullYear()} <br>
```
Comment (TueeNguyen) [inline src/docs/docusaurus.config.js:116]: Nice!

**Review by TueeNguyen — APPROVED** [review]

**Review by cindyledev — APPROVED** [review]
Fantastic work! Thank you so much

**Pull request merged**